frontend/src/components/ProductItem/ProductItem.js

```diff
@@ -1,27 +1,41 @@
 import React from 'react';
 
-import businessImage from "../../assets/business.jpg";
-import plusIcon from "../../assets/plus.svg";
+import { bakeryImages } from '../../shared/businessImages'
 
-const ProductItem = ({ products }) => (
-  products.map(product =>
-    <article id={product} key={'product-item ' + product} className="Product-item">
-      <h3 className="high-emphasis product-heading text-capitalize">{product}</h3>
+import roundedPlusIcon from "../../assets/rounded_plus.svg";
+import penIcon from "../../assets/pen.svg";
 
-      {products.map(product =>
-      <div key={'product-section ' + product} className="Product-section">
-        <img className="Product-img" src={businessImage} alt="product image" />
-        <div className="Product-content">
-          <div className="Product-review">
-            <h4>Very great bread indeed</h4>
-            <p>mix of some flourish stuff and water, plus some salty crystals looking like salt</p>
+const ProductItem = ({ type, products }) => {
+  const prices = [5.60, 3.45, 3.45, 3.45, 3.45];
+  const variants = bakeryImages
+    .map((eachBackeryImage, index) => ({ image: eachBackeryImage, price: prices[index] }));
+
+  const productIcons = {
+    add: roundedPlusIcon,
+    edit: penIcon,
+  }
+
+  return (
+    products.map(product =>
+      <article id={product} key={'product-item ' + product} className="Product-item">
+        <h3 className="high-emphasis product-heading text-capitalize">{product}</h3>
+  
+        {variants.map(variant =>
+        <div key={'product-section ' + variant.price} className="Product-section">
```

Comment:
I just noticed this now, sorry. Can you either:

1.  rename this component to something like `ProductGroup` and extract the Code inside this `.map` into a `<ProductItem />` Component
2.  Move the JSX surrounding this `.map` into `ProductList` and the code that is currently there into the view or a new component